website/docs/Installation.md

```diff
@@ -4,23 +4,31 @@
 
 When not using a docker container, we recommend using a virtual environment to install AutoGen. This will ensure that the dependencies for AutoGen are isolated from the rest of your system.
 
+Using Venv
 ```bash
 python3 -m venv autogen
 source autogen/bin/activate
 ```
 
+OR using conda
+```bash
+conda create -n autogen python=3.8
+conda activate autogen
+```
+
 ## Python
 
 AutoGen requires **Python version >= 3.8**. It can be installed from pip:
 
+Using pip
 ```bash
 pip install pyautogen
 ```
-<!--
+
 or conda:
-```
-conda install pyautogen -c conda-forge
-``` -->
+```bash
+conda install -c conda-forge pyautogui
```

Comment:
this looks like an error

Reply:
tried in conda , works
![image](https://github.com/microsoft/autogen/assets/60027013/a7b8f57d-444f-4305-b4ce-1d545e29b2c5)
![image](https://github.com/microsoft/autogen/assets/60027013/6907ba0b-969f-4b8e-910f-5638436ce874)

Reply:
`pyautogui` is a different package from pypi: https://pypi.org/project/PyAutoGUI/